guiEditor/src/components/sceneExplorer/treeItemComponent.tsx

```diff
@@ -194,18 +194,11 @@ export class TreeItemComponent extends React.Component<ITreeItemComponentProps,
             )
         }
 
-        const sortedItems = Tools.SortAndFilter(null, items);
-
+        let sortedItems = Tools.SortAndFilter(null, items);
+        sortedItems = sortedItems[0].getChildren();

```

Comment:
Why are you re-assigning? You can set it to const and chain everything, of course with making sure it doesn't throw